docs/install.rst

```diff
@@ -92,7 +92,10 @@ The install process will take some time, and it will return
 the terminal to you when it is complete. If an error occurs while
 running the install, please submit a detailed `GitHub
 issue <https://github.com/freedomofpress/securedrop/issues/new>`__ or
-send an email to securedrop@freedom.press.
+send an email to securedrop@freedom.press. You can also attempt to rerun
+the Ansible playbooks if you had a minor issue, such as a typo, after making
+edits to the file that are read from ``/install_files/ansible_base/prod_specific.yml``.
```

Comment:
I think you want to cover two cases here, the case of an admin making a mistake during `sdconfig`, and the case of Tor being flaky (or some other intermittent error), in which case the admin would want documentation to:

1. Describe that they can manually edit `install_files/ansible-base/site-specific/group_vars/all/site-specific` (used instead of `prod-specific.yml`) _or_ re-run `./securedrop-admin sdconfig --force` to retype all vars in `site-specific`.

2.  Describe that it's safe to rerun `./securedrop-admin install` if they see an error and let them know they aren't going to break anything (this closes #2098).

Reply:
Also let's put this troubleshooting above "make an issue or email us" :-)